internal/cmd/skupper/link/kube/link_generate_test.go

```diff
@@ -707,7 +707,7 @@ func TestCmdLinkGenerate_Run(t *testing.T) {
 					},
 				}
 			},
-			errorMessage: "the active site has not configured endpoints yet",
+			errorMessage: "A link cannot be generated because link access is not enabled. \n Use \"skupper site update --enable-link-access\" to enable it.",
 		},
 		{
 			name: "runs fails because there are no active site",
```

Comment:
```suggestion
			name: "Link generate fails because there are no active site",
```
I know it's not part of this PR, but caught my eye, there are a few instances of this. Maybe it's only used for testing?

Reply:
Yes, it is just the name of the test that the `Run()` function; but I agree with you that it needed some corrections.